packages/effect/src/Schema.ts

```diff
@@ -5782,6 +5782,42 @@ export const StringFromHex: Schema<string> = makeEncodingTransformation(
   Encoding.encodeHex
 )
 
+/**
+ * Decodes a URI component encoded string into a UTF-8 string.
+ *
+ * @example
+ * import { Schema } from "effect"
+ *
+ * const encoded = Schema.encodeSync(Schema.StringFromUriComponent)("hello world")
+ *
+ * const searchParams = new URLSearchParams()
+ * searchParams.append("hello", encoded)
+ *
+ * router.push(searchParams.toString()) // router.push("?hello=hello%20world")
```

Comment:
The examples are type-checked and run by our CI (which is failing because `router` doesn't exist). IMO you can simply `console.log` the output here

Reply:
ohhh ok thansk !